modules/ROOT/pages/using-opscenter.adoc

```diff
@@ -33,6 +33,15 @@ cat /var/log/rtf-init.log | grep -B 1 -A 3 "Ops Center"
 [NOTE]
 Verify port 32009 is open and accessible in your Security Group, Network Security Group or firewall rules. If using a private IP address, ensure you have access to the same internal network as the VM resides in.
 
+== Regain Access to the Ops Center
+If you lose the install logs, perform the following steps:
+
+. SSH into one of the nodes of the Runtime Fabric installation.
```

Comment:
May be able to simplify: "Open a terminal into one of the controller nodes."

Reply:
Updated.

Reply:
Got it - thanks!